src/App.js

```diff
@@ -1,5 +1,105 @@
+import { Console } from "@woowacourse/mission-utils";
+import { Random } from "@woowacourse/mission-utils";
```

Comment:
Airbnb 스타일 가이드에 따르면 같은 경로에서 여러 줄로 import하기보다 한 줄로 작성하는 것을 권장합니다.

[Airbnb JavaScript Style Guide : no-duplicate-imports](https://github.com/airbnb/javascript?tab=readme-ov-file#modules--no-duplicate-imports)
```suggestion
import { Console, Random } from "@woowacourse/mission-utils";
```